src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -735,9 +749,10 @@ public function testMultiLineStringLastResortParsing()
   You can have things that don't look like strings here
   true
   yes you can
+  with a: colon
```

Comment:
This does not seem to be allowed. Does it not trigger the deprecation?

Reply:
No it doesn't here because in the check I also check if the line is indented

Reply:
Then the indentation handling looks wrong to me. Afaik this is not valid YAML.

Reply:
I'll simplify the check. If we are dealing with a multiline string, whenever we encounter ': ' a deprecation notice would be thrown.